packages/react-instantsearch/src/widgets/CurrentFilters/index.js

```diff
@@ -1,8 +1,25 @@
+/**
+ * @module widget/CurrentFilters
+ */
 import connect from './connect';
-import CurrentFilters from './CurrentFilters';
+import CurrentFiltersComponent from './CurrentFilters';
```

Comment:
Could we say that any capitalized token is a Component otherwise it's just a function/HOC?

I am not sure suffixing Component everywhere brings more context/value to the code?

For sure we need to understand that some component are connected, some are not but do we actually need to express it everywhere?